setup.py

```diff
@@ -52,7 +50,7 @@
         package_dir={
             '': 'lib',
         },
-        scripts=['ec2deprecateimg'],
+        scripts=['ec2deprecateimg', 'ec2publishimg'],
         classifiers=[
             'Development Status :: 5 - Production/Stable',
             'Intended Audience :: Developers',
```

Comment:
Outside of review. But we can drop pythn3.3 as supported since it no longer is a supported Python version.

Reply:
I'll try and remember to pick that up with the next merge